### PR TITLE
Release/1.0.0 beta.3

### PR DESCRIPTION
**Pull request by perruche**
Release is containing :
- #23 
- #24 
- #25 
- #26

### CODE REVIEW

File: template/.gitlab-ci.yml [hunk 1]
```diff
@@ -1,46 +1,76 @@
...
+        - tailwind.config.js
+        - .gitlab-ci.yml
+        - web/wp-content/themes/**/*.{js,scss,vue}
+
+.Composer default config: &composer_default_config
+  image: php:7.2
+  before_script:
+    - apt-get update
+    - apt-get install -y git wget unzip zip
+    - wget https://raw.githubusercontent.com/composer/getcomposer.org/6e874537c7ff87f03f99dc8dea1f2c48e4274da3/web/installer -O - -q | php -- --quiet
```
Comment (titouanmathis) [inline template/.gitlab-ci.yml]: We need to add the version for the Composer installation, we are not ready for Composer 2.

```bash
wget https://raw.githubusercontent.com/composer/getcomposer.org/6e874537c7ff87f03f99dc8dea1f2c48e4274da3/web/installer -O - -q | php -- --quiet --version=1.10.16
```

**Review by titouanmathis — CHANGES_REQUESTED** [review]
A small fix for the Composer installation and a question: why not including #22 in the release?

**Comment by perruche** [comment]
> A small fix for the Composer installation and a question: why not including #22 in the release?

#22 is in the release, target branch was not `develop`, i cannot change it anymore 😞

**Review by titouanmathis — APPROVED** [review]
👍 

Let's get this released!

**Pull request merged**